spring/src/main/java/org/axonframework/spring/authorization/UnauthorizedCommandException.java

```diff
@@ -0,0 +1,18 @@
+package org.axonframework.spring.authorization;
+
+/**
+ * Exception indicating that a command has been rejected due to a lack of authorization.
+ *
+ * @author Roald Bankras
```

Comment:
Nit: please add `@since 4.10.0` here:

```suggestion
 * @author Roald Bankras
 * @since 4.10.0
```